client/my-sites/customer-home/main.jsx

```diff
@@ -361,105 +463,26 @@ class Home extends Component {
 					{ // "Go Mobile" has the highest priority placement when viewed in smaller viewports, so folks
 					// can see it on their phone without needing to scroll.
 					isMobile() && <GoMobileCard /> }
-					{ displayChecklist ? (
+					{ displayChecklist && (
```

Comment:
We only want to show Site Tools after the checklist has been completed; with this change, the Site Tools display as an empty card underneath the checklist:

<img width="1100" alt="Screen Shot 2020-01-28 at 4 39 48 PM" src="https://user-images.githubusercontent.com/2124984/73307812-f06d3e80-41ec-11ea-9905-a535a9c0cdfb.png">

The logic should look like:

```
if ( displayChecklist ) { 
    //show the checklist
} else {
    //show site tools
}
```

Reply:
Great catch @sixhours !